LICENSE-MPL-RabbitMQ

```diff
@@ -327,7 +327,7 @@
      shall survive.
 
      8.2.  If You initiate litigation by asserting a patent infringement
-     claim (excluding declatory judgment actions) against Initial Developer
+     claim (excluding declaratory judgment actions) against Initial Developer
```

Comment:
Oh, I should call this one out. This is a license file, lawyers typically have strong opinions about them. Generally, dictionaries prefer the spelling I'm suggesting. But, I'd certainly understand people objecting, and dropping this change (or any other) isn't a problem.

Reply:
👍